admin/RunExport

```diff
@@ -55,6 +54,16 @@ chown "$FTP_USER:$FTP_GROUP" \
     "$FTP_DATA_DIR"/replication \
     "$FTP_DATA_DIR"/replication/{daily,weekly}
 
+# Create temporary directory under the same volume as the final
+# destination so that moving files does not require copying those.
+# If we're not doing a full export, just use default tmp instead.
```

Comment:
1. The default temporary directory usually has better performances.
2. It would make assumptions from `admin/RunExport` on `admin/ExportAllTables`.
    When `WITH_FULL_EXPORT` is set to `1`, their final destination under `$FTP_DATA_DIR` is known in `admin/RunExport`.
    In the opposite case (when `WITH_FULL_EXPORT` is set to `0`), it isn’t clear what happens. I guess that it is used to make replication packets only, and these seem to be written under this temporary directory too. But I didn’t find what happens next.

Reply:
OK, I guess it doesn't matter much since the packets aren't large to begin with.